### PR TITLE
v3.0.1

### PR DESCRIPTION
**Pull request by Gudahtt**
Updates the version to v3.0.1 and updates the changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.1] - 2020-09-11
+
+### Fixed
+- Ensure balances >= 0.1 and < 1 start with a zero #(47)
```
Comment (whymarrh) [inline CHANGELOG.md:12]:
```suggestion
- Ensure balances >= 0.1 and < 1 start with a zero (#47)
```

**Review by whymarrh — CHANGES_REQUESTED** [review]

**Comment by whymarrh** [comment]
Also the links will need to be updated

File: CHANGELOG.md (hunk 1 above)
Comment (Gudahtt) [reply CHANGELOG.md]: Whoops! Thanks

**Comment by Gudahtt** [comment]
Which links?

**Comment by whymarrh** [comment]
> Which links?

Oh, there aren't any. We can fix that later.

**Review by whymarrh — APPROVED** [review]

**Comment by whymarrh** [comment]
> A new version of the package `@metamask/eth-token-tracker` (3.0.1) was published at 2020-09-11T15:51:55.387Z from
[redacted]. The shasum of this package was `36b89f39c2af2ac41aa600dcedd2341704d94484`.

**Pull request merged**